src/components/Login.js

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
-import { browserHistory } from 'react-router';
-import './stylesheets/Login.css';
-import cx from 'classnames';
+import { hashHistory, browserHistory } from 'react-router';
+import '../stylesheets/Login.scss';
 
 class Login extends React.Component {
   handleLogin = (e) => {
     e.preventDefault();
     if (this.refs.email.value && this.refs.password.value) {
-      browserHistory.push("play");
+      browserHistory.push('play');
+    }
+  }
+
+  handleLogin = (e) => {
+    e.preventDefault();
+    if (this.refs.username.value && this.refs.password.value) {
+      hashHistory.push('play');
```

Comment:
Why are there two copies? We should also be using browserHistory rather than hash no?